src/MaaWpfGui/Res/Localizations/en-us.xaml

```diff
@@ -345,7 +345,7 @@ Please do not check it when the stage OF-1 is not unlocked.</system:String>
     <system:String x:Key="PromptRestartForSettingsChange">Restart immediately to apply changes?</system:String>
     <system:String x:Key="SwitchResolutionTip">Change the emulator resolution to 1920x1080!</system:String>
     <system:String x:Key="ResolutionInfoYoStarEN">Make sure the emulator is set to 1920x1080</system:String>
-    <system:String x:Key="AllowUseStoneSave">Save &quot;Use Originium&quot; Setting</system:String>
+    <system:String x:Key="AllowUseStoneSave">Save "Use Originium" Setting</system:String>
```

Comment:
I will never stop saying this. 
We need more guidelines in webplate. Inserting `"`, `<`, `>` will break the file, leading to build errors